esphome/components/ina226/sensor.py

```diff
@@ -92,7 +93,8 @@ def validate_adc_time(value):
             cv.Optional(CONF_MAX_CURRENT, default=3.2): cv.All(
                 cv.current, cv.Range(min=0.0)
             ),
-            cv.Optional(CONF_ADC_TIME, default="1100 us"): validate_adc_time,
+            cv.Optional(CONF_ADC_TIME_VOLTAGE, default="1100 us"): validate_adc_time,
+            cv.Optional(CONF_ADC_TIME_CURRENT, default="1100 us"): validate_adc_time,
```

Comment:
Something like this would make the change non-breaking and allow people to specify a single time for both, or split the times if they want to.
```suggestion
            cv.Optional(CONF_ADC_TIME, default="1100 us"): cv.Any(
                validate_adc_time,
                cv.Schema(
                    {
                        cv.Required(CONF_VOLTAGE): validate_adc_time,
                        cv.Required(CONF_CURRENT): validate_adc_time,
                    }
                ),
            )
```

Reply:
I'm not sure how this works...
Would it result in a yaml config like this?

adc_time: 140us
or optionally:
adc_time:
  voltage: 140us
  current: 332us

Reply:
OK cool, thanks for that suggestion.